addons/web/static/src/webclient/actions/client_actions.js

```diff
@@ -48,25 +48,33 @@ registry.category("actions").add("invalid_action", InvalidAction);
  * Client action to reload the whole interface.
  * If action.params.menu_id, it opens the given menu entry.
  * If action.params.action_id, it opens the given action.
- * If action.params.wait, reload will wait the server to be reachable before reloading
  */
 function reload(env, action) {
-    const { menu_id, action_id, wait } = action.params || {};
```

Comment:
is it the right commit to remove the wait param? I think it is still used. I would have removed it in a later commit alongside the change of the two actions using it (+ the removal of router.redirect)